pkg/k8sutil/operator.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package k8sutil
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func RestartDeployment(namespace, deploymentName string) {
+	// Get Kubernetes Client
+	_, clientset, _ := GetKubeClient()
+
+	deploymentsClient := clientset.AppsV1().Deployments(namespace)
+	data := fmt.Sprintf(`{"spec": {"template": {"metadata": {"annotations": {"kubectl.kubernetes.io/restartedAt": "%s"}}}}}`, time.Now().String())
+	_, err := deploymentsClient.Patch(context.TODO(), deploymentName, types.StrategicMergePatchType, []byte(data), v1.PatchOptions{})
```

Comment:
To restart the operator, I thought we could delete the pod, and using labels to match the pod. Patching this deployment actually restarts the operator?

Reply:
yes, in my local testing it was restarting, I share the output today. 
We have 3ways to restart

1. using scale --replica=0
2. patch
3. delete

Reply:
![Screenshot from 2023-01-25 11-37-54](https://user-images.githubusercontent.com/34444611/214492866-fc2366e6-b63e-4521-b9e4-97f4d199062a.png)

Reply:
we can see that operator has restarted using the above command. Merging now.